bitcoin/src/taproot.rs

```diff
@@ -481,20 +459,17 @@ impl TaprootBuilder {
     ) -> Result<TaprootSpendInfo, TaprootBuilder> {
         match self.branch.len() {
             0 => Ok(TaprootSpendInfo::new_key_spend(secp, internal_key, None)),
-            1 => {
+            1 =>
                 if let Some(Some(node)) = self.branch.pop() {
                     Ok(TaprootSpendInfo::from_node_info(secp, internal_key, node))
                 } else {
                     unreachable!("Size checked above. Builder guarantees the last element is Some")
-                }
-            }
+                },
```

Comment:
I may be paranoid but this style seems to ask for heartbleed-style vulnerability.

Reply:
Heh, I agree it feels a bit treacherous, but Rust will at least require you put **something** (not just whitespace) before any following lines of code, so you can't be easily fooled into mistaking which block it belongs to.